### PR TITLE
6562 updating heron-admin spin up in dev mode

### PR DESCRIPTION
**Pull request by anuforok**
missed `/`in the 0c609b4
Screen Shot 2021-06-11 at 2 15 41 PM

 after fixing above error in 0c609b4, the below errros not setting up the directories right, eg index.html
Screen Shot 2021-06-11 at 12 46 14 PM
Fixes include:- stylesheet link was not properly ended in the py:match section and also did not specify the async tag for python2 properly.

### CODE REVIEW

File: heron_wsgi/templates/kumc_layout.xml [hunk 1]
```diff
@@ -3,28 +3,17 @@
 <py:match path="head" once="true">
    ${select('*')}
 
-  <link rel="stylesheet" type="text/css" href=
-  "https://www.kumc.edu/css/template2009/ku-sec.css" />
+  <link rel="stylesheet" type="text/css" href="https://www.kumc.edu/css/template2009/ku-sec.css" />
```
Comment (nihr43) [inline heron_wsgi/templates/kumc_layout.xml:6]: i dont see the intention of these particular changes.  was this causing a problem when trying to run the software locally or is this unrelated?

**Comment by nihr43** [comment]
can you add a description?  is that a screenshot of before or after?

File: heron_wsgi/templates/kumc_layout.xml (hunk 1 above)
Comment (anuforok) [reply heron_wsgi/templates/kumc_layout.xml:6]: Dear @nihr43 this was just me formatting the code to trace a tag that was not properly ended from my code update and also reducing lines of code.

**Comment by nihr43** [comment]
and will you rebase and squash the empty commits e02d68e, cc8f77a

**Comment by lav-patel** [comment]
Looks good to me. Suggest to make code refactoring in separate branch.

**Pull request merged**